src/sage/schemes/affine/affine_morphism.py

```diff
@@ -355,7 +347,7 @@ def _fastpolys(self):
 
             sage: P.<x,y> = AffineSpace(QQ, 2)
             sage: H = Hom(P, P)
-            sage: f = H([x^2+y^2, y^2/(1+x)])
+            sage: f = H([x^2 + y^2, y^2/(1+x)])
```

Comment:
needs spaces around the second `+`.

Reply:
I'm not adding spaces within nested sums